README.md

```diff
@@ -116,6 +116,35 @@ dependencies {
 }
 ```
 
+### Specifying a Checker Framework version
+
+To change the version of the Checker Framework used by the plugin (by default version 2.8.0),
```

Comment:
I would update this to 2.8.1.  That also requires changing the example below to an older version.

Also, add a comment to line 20 of CheckerPlugin.groovy saying that whenever that line is updated, this line of the README.md file needs to be updated.